lnwallet/channel.go

```diff
@@ -514,6 +515,12 @@ type commitment struct {
 	// chain is being modified, the opposite is true.
 	txn *wire.MsgTx
 
+	// outUnknowns is a slice containing matching derivation informfation to
+	// the outputs of the transaction in txn, used to pass to any downstream
+	// signer that can use the information to ensure the channel state
+	// machine is behaving correctly.
+	outUnknowns [][]*psbt.Unknown
```

Comment:
In another project we created an alias type for `psbt.Unknown`. Since these values aren't really unknown to this code, we could also call the variables differently (e.g. "meta" or "signInfo"). I know that would require a big rename in this PR, so it's really only a light suggestion at this point.